src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -12,6 +12,31 @@
              mc:Ignorable="d">
 
     <UserControl.Resources>
+        <Style x:Key="ZoomRepeatButtonStyle" TargetType="RepeatButton">
+            <Setter Property="Width" Value="36"/>
+            <Setter Property="Height" Value="36"/>
+            <Setter Property="CornerRadius" Value="18"/>
+            <Setter Property="Background" Value="White"/>
+            <Setter Property="BorderThickness" Value="1"/>
+            <Setter Property="FontSize" Value="12"/>
+            <Setter Property="Delay" Value="500"/>
+            <Setter Property="Interval" Value="40"/>
+        </Style>
+        <Style x:Key="ZoomButtonStyle" TargetType="Button">
+            <Setter Property="Width" Value="36"/>
+            <Setter Property="Height" Value="36"/>
+            <Setter Property="CornerRadius" Value="18"/>
+            <Setter Property="Background" Value="White"/>
+            <Setter Property="BorderThickness" Value="1"/>
+            <Setter Property="FontSize" Value="12"/>
+        </Style>
+        <Style x:Key="ActionButtonStyle" TargetType="Button">
+            <Setter Property="Width" Value="36"/>
+            <Setter Property="Height" Value="36"/>
+            <Setter Property="Background" Value="White"/>
```

Comment:
The buttons do not resize with the window size of calculator. I think there should be some visual states added to keep the buttons in proportion to the window size.

Reply:
Steph and I discussed offline. We still need to add visual states to make either the graph or equation area take the full screen when in a small window size. We can handle resizing the buttons at that point with those new visual states if they still need adjustment. That can be done in another PR.

We also discussed that this PR should add in the high contrast/narrator pieces mentioned in other comments.